backend/app/controllers/enrollments_controller.rb

```diff
@@ -71,6 +73,15 @@ def index
       root: "enrollments"
   end
 
+  # GET /enrollments/export
+  def export
+    @enrollments = policy_scope(Enrollment)
+
+    respond_to do |format|
+      format.csv { send_data @enrollments.to_csv, filename: "export-datapass-#{Date.today}.csv" }
+    end
```

Comment:
```suggestion
    send_data @enrollments.to_csv, type: 'text/csv', disposition: 'attachment', filename: "export-datapass-#{Date.today}.csv"
```

1. Simplify according to my comment on the module 
2. Add type to force browser to download it (for old browser versions)
3. Specify to the browser to act as a downloadable link, which keeps the actual page open (more convenient)